<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label_annotations">
     <property name="text">
      <string>Annotations</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_variables">
     <property name="text">
      <string>Variables</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit_name"/>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEdit_unique_id"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_kind">
     <property name="text">
      <string>Kind</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QTableWidget" name="tableWidget_annotations">
     <column>
      <property name="text">
       <string>Key</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QTableWidget" name="tableWidget_variables">
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>Variable</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="lineEdit_kind"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
